tests/test_cli.py

```diff
@@ -189,6 +190,11 @@ def test_pipeline_ae(self, config, callback_values):
         config.ae.userid_column_name = "user_col"
         config.ae.userid_filter = "user321"
 
+        with open(os.path.join(TEST_DIRS.data_dir, 'columns_ae.txt')) as fh:
+            expected_columns = [line.strip() for line in fh]
```

Comment:
We use this three times. should we move it in to a helper function?
```python
def read_lines_from_test_data(filename):
    with open(os.path.join(TEST_DIRS.data_dir, filename)) as fh:
            expected_columns = [line.strip() for line in fh]
```
```python
expected_columns = read_lines_from_test_data("columns_ae.txt")
```

Reply:
done